src/cdk/a11y/focus-monitor/focus-monitor.ts

```diff
@@ -134,7 +134,7 @@ export class FocusMonitor implements OnDestroy {
     // Make a note of when the window regains focus, so we can
     // restore the origin info for the focused element.
     this._windowFocused = true;
-    this._windowFocusTimeoutId = setTimeout(() => (this._windowFocused = false));
+    this._windowFocusTimeoutId = window.setTimeout(() => (this._windowFocused = false));
```

Comment:
Can we get around that by disabling automatic type resolution in the tsconfig? i.e setting `types: []`?

Reply:
I tried that, but it didn't seem to work. BTW, the typing problem only happens on Windows, so it's probably related to the runfile problem.

Reply:
Maybe it's related to microsoft/TypeScript#37708. The vanilla `tsc_wrapped` from Bazel must have done something special to avoid this problem.

Reply:
I briefly looked into it and tested locally. I think the issue is that the `tsec` macro pulls in all deps as `files` in the tsconfig. I don't think this is necessarily matching with what `ng_module`/`ts_library` does. 

I don't think it's necessarily worth fixing though. Just wanted to provide some more context. I'm good proceeding as there are very few instances.